packages/ir-sdk/fern/apis/ir-types-v60/changelog/CHANGELOG.md

```diff
@@ -0,0 +1,404 @@
+# Changelog
```

Comment:
We omit the changelog directory when cataloging IR versions

Reply:
fixing